core/domain/exp_jobs_one_off_test.py

```diff
@@ -526,7 +526,7 @@ def test_reverts_not_counted(self):
 
         user_a_id = self.get_user_id_from_email(self.EMAIL_A)
 
-        # Let USER A make 2 non-revert commits and 1 revert
+        # Let USER A make 3 non-revert commits and 1 revert
```

Comment:
OK, let's drop it. Usually comments in tests apply to the next block only, and the duplication here doesn't really make sense...